doc/LOGGING.md

```diff
@@ -77,3 +77,120 @@ See below an example of what the emitted metrics may look like in the logs.
 
 We recommend using the metrics only for debugging at this time.
 Metrics are currently output in an unstructured format and are subject to change in future releases.
+
+### Publishing Metrics
+
+Additionally, metrics can be published to a specified endpoint (in OpenTelemetry format), for example to the endpoint that CloudWatch agent listens to and can be configured to forward metrics to CloudWatch/Prometheus. 
```

Comment:
I think we can make this first paragraph more intuitive by separating publication and integration. Maybe something like:

> Additionally, metrics can be published in OpenTelemetry format to a specified endpoint. This allows for integration with (generic term of destination. OPTL agents? idk), for example CloudWatch or Prometheus.

CloudWatch could be a link to the section which includes details and links to documentation (instead of a paragraph here in the introduction).